vagrant/provisioning/roles/ldap/tasks/main.yml

```diff
@@ -73,7 +90,42 @@
       samAccountName: "{{ item.name }}"
       cn: "{{ item.name }}"
   loop: "{{ ldap_groups|flatten }}"
-     
+
+- name: remove portal group from ArkCase OU if exists
+  ldap_entry:
+    server_uri: "{{ ldap_url }}"
+    validate_certs: no
+    bind_dn: "{{ ldap_bind_user }}"
+    bind_pw: "{{ ldap_bind_password }}"
+    dn: "CN={{ item.name }},{{ ldap_group_no_base }},{{ ldap_base }}"
+    state: absent
+    objectClass:
+      - group
```

Comment:
What will happen to any users who might be in this group that you are removing?  Shall we first add the new portal group, and ensure to add any members from this old portal group, and then remove it?